app/controllers/admin/display_ads_controller.rb

```diff
@@ -28,10 +28,18 @@ def create
 
       if @display_ad.save
         flash[:success] = "Display Ad has been created!"
-        redirect_to admin_display_ads_path
+        if FeatureFlag.enabled?(:admin_restructure)
+          redirect_to admin_customization_display_ads_path
+        else
+          redirect_to admin_display_ads_path
+        end
       else
         flash[:danger] = @display_ad.errors_as_sentence
-        render new_admin_display_ad_path
+        if FeatureFlag.enabled?(:admin_restructure)
+          redirect_to new_admin_customization_display_ads_path
```

Comment:
```suggestion
          redirect_to new_admin_customization_display_ad_path
```
Same singular suggestion here! 😊

Reply:
oops just added a separate commit for this one. thanks!